tests/generators/test_dorogovtsev_goltsev_mendes.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import rustworkx
+
+
+class TestDorogovtsevGoltsevMendesGraph(unittest.TestCase):
+    def test_dorogovtsev_goltsev_mendes_graph(self):
+        for n in range(0, 6):
```

Comment:
Use subtests like in https://github.com/Qiskit/rustworkx/blob/8af19f0f9c3f80463e39bb691b8f13e96a5d243f/tests/graph/test_subgraph_isomorphic.py#L282

Reply:
Should it be one subTest per assertion, or just one for each n?

Reply:
Subtest for each N. The assertions go in the same subtest